packages/muelu/test/viz/Viz.cpp

```diff
@@ -294,7 +294,7 @@ int main_(Teuchos::CommandLineProcessor &clp, Xpetra::UnderlyingLib& lib, int ar
       }
 
       // AggregationExportFactory can't work with
-      // Aggregates_kokkos (yet)
+      // Aggregates (yet)
```

Comment:
delete comment?

Reply:
what about the `mueluList.set("use kokkos refactor", false);` right below it?

Reply:
right. I'll wait to crack down on the weird uses of `"use kokkos refactor"` until we merge everything then :)